addons/l10n_es_pos/views/pos_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" ?>
+<odoo>
+    <!-- POS order form -->
+    <record id="view_pos_pos_form_simplified_invoice" model="ir.ui.view">
+        <field name="name">pos.order.form</field>
+        <field name="model">pos.order</field>
+        <field name="inherit_id" ref="point_of_sale.view_pos_pos_form" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="l10n_es_unique_id" readonly="1" />
```

Comment:
shouldn't this be hidden if the company is not spanish ? It's a real question I do not know what is the rule we follow in the point of sale for localization field but I would find it strange if we show fields that are localization specific to all company

Reply:
And same for the other views

Reply:
following our talk, we decided to not hide one of the views for now.